### DISCUSSION TITLE
i²c connector on 3.01 Opendesk

### DISCUSSION THREAD

**Discussion by gh0stless** [shanteacontrols/OpenDeck#184] [General]
i²c connector on 3.01 Opendesk hardware is labeled incorrectly. Maybe write a hint in the wiki?

**Comment by paradajz** [comment]
It's called **OpenDeck**, not OpenDesk. And yes, I've noticed this when I was designing v3.1 board but forgot about it in the meantime. Will update the wiki soon.

**Comment by paradajz** [comment]
I've updated the wiki page.
